src/input/touchmouse.js

```diff
@@ -26,19 +33,13 @@ inherit(TouchMouseInput, Input, {
         var isTouch = (inputData.pointerType == INPUT_TYPE_TOUCH),
             isMouse = (inputData.pointerType == INPUT_TYPE_MOUSE);
```

Comment:
Better make it:
```javascript
if (isMouse && inputData.sourceCapabilites && inputData.sourceCapabilities.firesTouchEvents) { 
   return;
}
```
So that it doesn't throw on browsers that don't support `sourceCapabilities`.